benchmarks/overrides_benchmark/common.py

```diff
@@ -0,0 +1,16 @@
+import torch
+
+NUM_REPEATS = 1000
+NUM_REPEAT_OF_REPEATS = 1000
+
+
+class SubTensor(torch.Tensor):
+    pass
+
+
+class WithTorchFunction(torch.Tensor):
+    def __torch_function__(self, func, args=(), kwargs=None):
+        if kwargs is None:
+            kwargs = {}
+
+        return args[0] + args[1]
```

Comment:
Maybe also a test with a tensor-like class that doesn't inherit from `torch.Tensor` and implements `__torch_function__`.

Reply:
Done.